#! /bin/sh /usr/share/dpatch/dpatch-run
## 002-148-directfb.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make 1.4.8 compile with directfb again

@DPATCH@
--- cairo-1.4.8/src/cairo-directfb-surface.c.orig	2007-06-21 00:06:16.000000000 -0700
+++ cairo-1.4.8/src/cairo-directfb-surface.c	2007-06-21 00:06:49.000000000 -0700
@@ -1572,7 +1572,7 @@
         NULL, /* show_glyphs */
 #endif
         NULL, /* snapshot */
-	_cairo_directfb_is_similar,
+	_cairo_directfb_surface_is_similar,
 	NULL /* reset */
 };
 
